mkosi

```diff
@@ -1814,6 +1835,7 @@ def parse_args():
     group.add_argument('-C', "--directory", help='Change to specified directory before doing anything', metavar='PATH')
     group.add_argument("--default", dest='default_path', help='Read configuration data from file', metavar='PATH')
     group.add_argument("--kernel-commandline", help='Set the kernel command line (only bootable images)')
+    group.add_argument("--hostname", help="Set hostname")
```

Comment:
I think the logic should be the following:
* if neither `--hostname` option nor `Hostname` config key are given, do nothing (leave whatever has been installed as per `/etc/hostname`);
* if either `--hostname` or `Hostname` are specified with an empty value, reset hostname (i.e. remove `/etc/hostname`);
* if a non-empty value is given, set the hostname via `/etc/hostname`.

@poettering WDYT?